tests/test_stamp.py

```diff
@@ -26,13 +28,12 @@ def test_stamp_mass_PI(T_A, T_B):
     trivial_idx = 2
     zone = int(np.ceil(m / 2))
     Q = T_B[trivial_idx : trivial_idx + m]
-    T_subseq_isconstant = core.rolling_isconstant(T_B, m)
```

Comment:
This line is removed as the purpose of this test function is not to test the support for isconstant param. Instead, I added new test functions at the end of the `tests/test_stamp.py`